src/core/AutoRest.Core/Properties/Resources.resx

```diff
@@ -241,6 +241,9 @@ Modelers:
   <data name="TrackedResourceListByImmediateParentOperationMissing" xml:space="preserve">
     <value>The child tracked resource, '{0}' with immediate parent '{1}', must have a list by immediate parent operation.</value>
   </data>
+  <data name="XMSPagableListByRGAndSubscriptionsMismatch" xml:space="preserve">
+    <value>For the tracked resource '{0}', the x-ms-pageable must be same for list by resource group and subscriptions operations.</value>
```

Comment:
sure, how about something like "For tracked resource '{0}', x-ms-pageable extension values must be the same for....". I would just thinking of clarifying what we mean by  "x-ms-pageable must be the same"